tests/conftest.py

```diff
@@ -86,8 +87,12 @@ def mockreturn(*args, **kwargs):
 
 @fixture
 def alice_transaction_obj(alice_pubkey):
-    return Transaction.create(owners_before=[alice_pubkey],
-                              owners_after=[alice_pubkey])
+    serial_number = b64encode(urandom(10), altchars=b'-_').decode()
```

Comment:
Creating a random value here shouldn't be necessary for a payload uuid; but if you like using it then 👍.